File: backend/app/api/health.py
"""GET /api/health — reports the app is up. No database access (issue #2)."""

from fastapi import APIRouter

from app import APP_NAME, __version__

router = APIRouter()


@router.get("/health")
def get_health() -> dict[str, str]:
    return {"name": APP_NAME, "version": __version__}

File: frontend/src/api/health.ts
// Matches app/api/health.py's response shape exactly (issue #2).
import { apiFetch } from "./client";

export interface HealthResponse {
  name: string;
  version: string;
}

export function getHealth(): Promise<HealthResponse> {
  return apiFetch<HealthResponse>("/health");
}
